devel/libbacktrace/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        ianlancetaylor libbacktrace 7939218
```

Comment:
This will be a disaster with versioning. It should be either a proper version or a date, not a hash.